pkg/secrets/passdriver/passdriver.go

```diff
@@ -51,15 +51,15 @@ func defaultDriverConfig() *driverConfig {
 			filepath.Join(home, ".local/share/gopass/stores/root"),
 		}
 		for _, path := range defaultLocations {
-			if stat, err := os.Stat(path); err != nil || stat.IsDir() {
+			if stat, err := os.Stat(path); err != nil || !stat.IsDir() {
 				continue
 			}
 			cfg.Root = path
 			bs, err := ioutil.ReadFile(filepath.Join(path, ".gpg-id"))
 			if err != nil {
 				continue
 			}
-			cfg.KeyID = string(bs)
+			cfg.KeyID = strings.Trim(string(bs), "\r\n")
```

Comment:
Would this look simpler if you used
string(bytes.Trim(bs,"\r\n"))

Reply:
changed.